src/srtctl/README.md

```
@@ -130,17 +136,120 @@ resources:
 
 `CUDA_VISIBLE_DEVICES` is automatically set per worker (e.g., `0,1,2,3` and `4,5,6,7`).
 
+### Rollup Stage
+
+After benchmark completion, the rollup stage consolidates all experiment data into
+a single `rollup.json` file for easy analysis:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)